evaluator.go

```diff
@@ -645,10 +692,47 @@ func (c *evalContext) addResult(value string) {
 	c.currentResult.Add(value)
 }
 
+func (c *evalContext) addSliceToResult(slice []interface{}) {
+	if c.currentResult.Cardinality()+len(slice) >= MaxResults {
+		panic(tooManyResults{})
```

Comment:
`recover()` section of func `evalRangeInplace` would see `tooManyResults` , and returns results. This is same behavior in `addResult()` function